### DISCUSSION TITLE
Join group using invite code

### DISCUSSION THREAD

**Discussion by azizmhmmdf** [devlikeapro/waha#517] [Ideas]
Hi,
I don't see the possibility to join a group using the invite code to WhatsApp in your document.
It would be great if you could enable it in the next version

**Comment by devlikepro** [comment]
We'll do!
https://github.com/devlikeapro/waha/issues/518

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
https://waha.devlike.pro/docs/how-to/groups/#join-group

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
